### PR TITLE
Install layout home

### PR DESCRIPTION
**Pull request by caovanson06112001**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/52376)

## WHAT (optional)
- set up layout home_page, cart, order, product

## HOW
- Add css, js for home_page, cart, order, product . And add gem redis

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)

trang chủ
image

tìm kiếm sách
image

sản phẩm
image

image

### CODE REVIEW

**Comment by caovanson06112001** [comment]
ready

File: app/views/shares/website/_banner.html.erb [hunk 1]
```diff
@@ -0,0 +1,163 @@
+<div class="page-header page-header-0">
+  <div class="page-block">
+    <div class="row m-0">
+      <div class="col-lg-2 col-xl-2 col-xxl-2 menu-code p-0">
+        <div>
+          <button type="button" class="btn btn-primary w-100">
+            <%= t "category.title"%>
```
Comment (anlvq-1058) [inline app/views/shares/website/_banner.html.erb:7]:
```suggestion
            <%= t "category.title" %>
```

File: app/views/shares/website/_banner.html.erb [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+          </button>
+        </div>
+        <ul class="category_menu">
+          <li>
+            <a href="">
+              cao văn sơn
+            </a>
```
Comment (anlvq-1058) [inline app/views/shares/website/_banner.html.erb:14]: Chỗ này để làm gì em

File: app/views/shares/website/_books.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      <%= t "top_book"%>
+    </h4>
+    <hr>
+  </div>
+</div>
+
+<div class="text-center">
+  <a href="#">
+    <button class="btn btn-outline-danger more-btn">
+      <%= t "more"%> &gt;&gt;&gt;
```
Comment (anlvq-1058) [inline app/views/shares/website/_books.html.erb:13]:
```suggestion
      <%= t "more" %> &gt;&gt;&gt;
```

File: app/views/shares/website/_books.html.erb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<div class="row">
+  <div class="col-sm-12">
+    <h4 class="m-43 text-center">
+      <%= t "top_book"%>
```
Comment (anlvq-1058) [inline app/views/shares/website/_books.html.erb:4]:
```suggestion
      <%= t "top_book" %>
```

**Review by VanHoa18042000 — APPROVED** [review]
approved

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by caovanson06112001** [comment]
ready

**Pull request merged**